### PR TITLE
(#395)Implement Search Criteria on Trial Results 

### LINKED ISSUE #395: Implement Search Criteria Table on Trial Results
The Search Criteria Table displays on the results page through the ResultsPageHeader.jsx component. This will need to be updated to pass in the search criteria and remove redux. 
* You will have to get the modify search link / start over logic from the old [SearchCriteriaTable](https://github.com/NCIOCPL/clinical-trials-search-app/blob/develop/src/components/atomic/SearchCriteriaTable/SearchCriteriaTable.jsx#L322)
* For sanity, the ResultsPageHeader should accept 2 methods:
  * onModifySearchClick
  * onStartOverClick
* Then move the modify and start over logic into the ResultPage handlers. (Including Analytics too)
   * Clicking the modify search link will need to send the search criteria to the AdvancedSearchPage (through a navigate() call)
      * The AdvancedSearchPage will need to be updated to replace the form state with the searchCriteria.
      * This should allow a lot of the redux stuff to be removed from the results page.
   * The start over link will need to navigate to the correct form.  The SearchPage will have to be updated to handle this logic.
      * The search criteria state should be empty indicating a new search can be performed.
      * The modify search link will need to be copied from the original search criteria

### Acceptance Criteria

```gherkin
Scenario: [title]
  Given [context]
    And [some more context]
  When  [event]
  Then  [outcome]
    And [another outcome]
```

### Resources:
* Feature: JIRA Ticket #
* Mockups: *Here goes a URL to or the name of the mockup(s) in inVision*
* Web Page: *Here goes a URL to an existing web page*

### Notes

*Some complementary notes if necessary:*

* > Here goes a quote from an email
* Here goes whatever useful information can exist…

---

### Prerequisites
- [x] #381
- [ ] #347

### LINKED ISSUE #434: Failed validation on proptype in Age textField
### Issue description
*Description of the issue*
TextInput of Age field is only allowing string, therefore throwing warning  

VM24782:85 Warning: Failed prop type: Invalid prop `value` of type `number` supplied to `TextInput`, expected `string`.

> **ESTIMATE** TBD

### Steps to reproduce the issue

1.  go to basic search 
2. enter age
3. hit search
4. hit modify search
5. see the warning in the console

### What's the expected result?
-

### What's the actual result?
-

### Additional details / screenshot
- Screenshot

### Related Tickets
* Issue #9999
* Issue #9999

### PR DESCRIPTION
**Pull request by kate-mashkina**
Closes #395 
Closes #434

### CODE REVIEW

File: src/views/ResultsPage/ResultsPageHeader.jsx [hunk 1]
```diff
@@ -67,46 +86,62 @@ const ResultsPageHeader = ({
...
-									onClick={() => handleReset(START_OVER_LINK)}>
-									Start Over
-								</Link>
-							</>
-						) : (
-							''
-						)}
-					</strong>
-				</div>
+				<>
```
Comment (arcepaul) [inline src/views/ResultsPage/ResultsPageHeader.jsx:89]: no fragment needed here

**Review by arcepaul — APPROVED** [review]

File: src/views/ResultsPage/ResultsPageHeader.jsx (hunk 1 above)
Comment (KateMashkinaNIH) [reply src/views/ResultsPage/ResultsPageHeader.jsx]: removed it

**Review by bryanpizzillo — CHANGES_REQUESTED** [review]
This needs Redux removed.

**Comment by sarinapadilla** [comment]
@kate-mashkina demo'd the Search Criteria Table functionality for both me and @VictoriaSunNIH and it's looking good, so this passes from a UX perspective!

**Review by bryanpizzillo — APPROVED** [review]

**Pull request merged**